spec/features/state_file/complete_intake_spec.rb

```diff
@@ -153,6 +153,12 @@
   end
 
   context "AZ", :flow_explorer_screenshot, js: true do
+    before do
+      # TODO: remove this stub when we update Rails.configuration.statefile_current_tax_year to 2024
+      allow(Rails.configuration).to receive(:statefile_current_tax_year).and_return 2024
+      stub_const('DateAccessible::TAX_YEAR', Date.new(2024))
+    end
+
```

Comment:
this gets nuked in the next PR https://github.com/codeforamerica/vita-min/pull/4994, pls ignore